### DISCUSSION TITLE
esp32/MicroPython/17.esp32-micropython-wifi

### DISCUSSION THREAD

**Discussion by giscus** [bigrich-luo/DocsGiscus#8] [General]
# esp32/MicroPython/17.esp32-micropython-wifi

https://docs.geeksman.com/esp32/MicroPython/17.esp32-micropython-wifi.html

**Comment by luoxihaha** [comment]
为什么一直..............就是连接不上

**Comment by Ya-chunJen** [comment]
ImportError: can't import name wifi_connect
